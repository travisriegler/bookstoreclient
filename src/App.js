import axios from 'axios';
import baseUrl from './config'

const App = () => {

  axios(`${baseUrl}/api/v1/books`).then(books => {
    console.log(books);
  })

  return <div>My first component</div>
}

export default App;