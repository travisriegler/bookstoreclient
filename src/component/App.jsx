import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import Layout from './layout/Layout';
import BookContainer from './book/BookContainer';
import Login from './user/Login';
import { SnackbarProvider } from 'notistack'
import Auth from './auth';
import Register from './user/Register';

const App = () => {

  return (
    <SnackbarProvider maxSnack={3}>
      <Router>
          <Layout>
            <Routes>
                <Route path="/login" element={<Login />} />
                <Route path="/register" element={<Register />} />
                <Route path="/" element={
                  <Auth>
                      <BookContainer />
                  </Auth>
                } />
            </Routes>
          </Layout> 
      </Router>
    </SnackbarProvider>
  )
}

export default App;