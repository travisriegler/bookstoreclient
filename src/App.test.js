import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App component", () => {
    it("Should render app without error", () => {
        const { asFragment } = render(<App />);
        expect(asFragment()).toMatchSnapshot();
    });
});