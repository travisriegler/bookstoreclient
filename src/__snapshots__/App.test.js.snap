// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Should render app without error 1`] = `
<DocumentFragment>
  <div>
    My first component
  </div>
</DocumentFragment>
`;
